spec/requests/portfolios_spec.rb

```diff
@@ -52,6 +52,24 @@
       end
     end
 
+    describe "POST admin tagged /portfolios/:portfolio_id/portfolio_items" do
+      let(:portfolio) { create(:portfolio) }
+      let(:portfolio_item) { create(:portfolio_item) }
+      let(:params) { {:portfolio_item_id => portfolio_item.id} }
+      before do
+        post "#{api}/portfolios/#{portfolio.id}/portfolio_items", :params => params, :headers => send("#{tag}_headers")
+      end
+
+      it 'returns a 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'returns the portfolio_item we just added to the portfolio' do
```

Comment:
Possibly change the language of this statement: along the lines of the portfolio_item includes a reference to the portfolio

Reply:
Done!